docs/guides/v2.md

```diff
@@ -39,9 +39,9 @@ v1 and v2 will both receive weekly releases occurring every Tuesday morning.
 | EmptyState                | View changes [here](#emptystate)       |
 | ExportModal               | No API changes.                        |
 | ExpressiveCard            | View changes [here](#expressivecard)   |
-| HTTPError403              | No API changes.                        |
-| HTTPError404              | No API changes.                        |
-| HTTPErrorOther            | No API changes.                        |
+| HTTPError403              | View changes [here](#fullpageerror)    |
+| HTTPError404              | View changes [here](#fullpageerror)    |
+| HTTPErrorOther            | View changes [here](#fullpageerror)    |
```

Comment:
Since HTTPError actually still exist also will be deprecated in the next version, maybe something more like ...

```suggestion
| HTTPError403              | No API changes. New version [here](#fullpageerror) |
| HTTPError404              | No API changes. New version [here](#fullpageerror) |
| HTTPErrorOther            | No API changes. New version [here](#fullpageerror) |
```